lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
```

Comment:
Bummer. If slack only sends us two parameters we should stick with the idea of only one update or one search per request. Parsing the text from the second param into more than one piece of information doesn't seem to good to me. And it will be easier for me to make the methods from the sheet.js module have single responsibilities (update only updates, search only searches).